test/resources/v4/program-orders.test.js

```diff
@@ -595,5 +595,124 @@ describe('RO.program.orders', () => {
         });
       });
     });
+    describe('#cancel', () => {
```

Comment:
Nit: separate the testing contexts with a blank line?
```suggestion

    describe('#cancel', () => {
```